### PR TITLE
add file2

### PR DESCRIPTION
**Pull request by YasserAbdellaoui**
i have added a file si hedi 
smd
yasser.

### CODE REVIEW

File: file2 [hunk 1]
```diff
@@ -0,0 +1 @@
+black man is here
```
Comment (vortexpotatoeee) [inline file2:1]: kahlouch zeyed

**Comment by vortexpotatoeee** [comment]
tahbatli

**Comment by YasserAbdellaoui** [comment]
how much

**Comment by vortexpotatoeee** [comment]
lel ekher

**Comment by YasserAbdellaoui** [comment]
daddyyy daddy chill 🙌🏿

**Comment by YasserAbdellaoui** [comment]
Screenshot 2024-02-13 093708

**Pull request merged**